src/app/components/stack-table/stack-table.component.html

```diff
@@ -0,0 +1,16 @@
+
+<h4>Flash Card Stacks</h4>
+<table>
+    <thead><th>ID</th><th>Stack Name</th><th>Description</th><th>Creator ID</th><th>Cards</th><th>Learners</th></thead>
+    <tbody>
+        <tr *ngFor ="let stack of stacks">
+            <td>{{stack.stackId}}</td>
+            <td>{{stack.name}}</td>
+            <td>{{stack.description}}</td>
+            <td>{{stack.creatorId}}</td>
+            <td>Cards</td>
```

Comment:
This page should display the complete list of cards for this stack as well. You can do this in a separate table.